### PR TITLE
Issue #75 Increase timeout for nightly job

### PR DESCRIPTION
**Pull request by amitkrout**
Increased the timeout by 1hr30mins. I have not done any special calculation for this, just increased by 1hr30mins to be on safer side for some more nightly/releases.

### CODE REVIEW

File: minishift-ci-index.yaml [hunk 1]
```diff
@@ -363,7 +363,7 @@
           git_repo: minishift
           ci_project: '{name}'
           ci_cmd: '/bin/bash centos_ci.sh'
-          timeout: '300m'
+          timeout: '390m'
```
Comment (praveenkumar) [inline minishift-ci-index.yaml]: @amitkrout, why you changed this, since now release job, is actually same as PR job and that doesn't timeout.

Comment (amitkrout) [reply minishift-ci-index.yaml]: Ahh..i misunderstood with nightly job. Thanks for pointing it out. Will revert it back

Comment (amitkrout) [reply minishift-ci-index.yaml]: Done

**Comment by LalatenduMohanty** [comment]
Tried to merge the PR but because of github issues it might not be merged.

**Comment by LalatenduMohanty** [comment]
Tried to merge the PR but because of github issues it might not be merged.

**Pull request merged**